catalog.html

```diff
@@ -87,219 +106,258 @@
         </ul>
       </nav>
       <h1 class="title">ПЕРФОРАТОРЫ</h1>
-      <div class="container-columns-catalog">
-        <section class="filter">
-          <h2 class="page-title">ФИЛЬТР:</h2>
-          <form class="filter-form" action="https://echo.htmlacademy.ru/" method="GET">
-            <fieldset>
-              <legend>ЦЕНА:</legend>
-              <div class="price-form">
-                <label class="visually-hidden" for="priсe-range-filter">Ценовой диапазон</label>
-                <input type="range" id="priсe-range-filter" min="0" max="30000" value="30000" />
+      <section class="filter">
+        <h2 class="page-title">ФИЛЬТР:</h2>
+        <form class="filter-form" action="https://echo.htmlacademy.ru/" method="GET">
+          <fieldset>
+            <legend>ЦЕНА:</legend>
+            <div class="range-filter">
+              <div class="range-controls">
+                <div class="scale">
+                  <div class="bar"></div>
+                </div>
+                <div class="toggle toggle-min" tabindex="0"></div>
+                <div class="toggle  toggle-max" tabindex="0"></div>
               </div>
-              <div class="price-form">
+              <div class="price-controls">
                 <label class="visually-hidden" for="min-priсe-filter">Минимальная цена</label>
```

Comment:
```suggestion
                <label class="visually-hidden" for="min-price-filter">Минимальная цена</label>
```

походу русские буквы были. Поправь.

Reply:
дальше все price поправь в этом блоке, они все с русской буквой `c`.
Тебе Vscode это почему-то не показывает, я на своём проверила, но потом, как поправишь, показыват, что изменения есть.